meinberlin/assets/scss/components/_item_detail_2.scss

```diff
@@ -52,4 +52,10 @@
 .item-detail-2__info-box {
     border: 1px solid $border-color;
     padding: 1em;
+    margin: 2 * $spacer 0 3.3 * $spacer;
```

Comment:
why use 3.3 here? it's good to keep our spacing as uniform as possible as makes everything look neater

Reply:
actually from the design it would be 3.375rem. shall i go with 3 or 3.5 then?

Reply:
3.5 is good :)

Reply:
actually I changed to 2 because the smalltitle has its own margin-top now (restored from how it was before).